pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -195,3 +207,34 @@ func (n *NetworkPolicyController) processClusterNetworkPolicy(cnp *secv1alpha1.C
 	}
 	return internalNetworkPolicy
 }
+
+// processRefCGs processes the ClusterGroup references present in the rules and returns the
+// NetworkPolicyPeer with the corresponding AddressGroups and IPBlocks.
+func (n *NetworkPolicyController) processRefCGs(cgs []string) ([]string, []controlplane.IPBlock) {
+	var ipb []controlplane.IPBlock
+	var ag []string
+	for _, g := range cgs {
+		// Retrieve ClusterGroup for corresponding entry in the rule.
+		cg, err := n.cgInformer.Lister().Get(g)
```

Comment:
Store `n.cgInformer.Lister()` as struct member `n.cgLister` to avoid creating an Informer and Lister every time a group is processed?

Reply:
done.. i already added the lister to struct.. but forgot to use it !